src/main/scala/nl/knaw/dans/easy/stage/Conf.scala

```diff
@@ -73,6 +73,10 @@ class Conf(args: Seq[String]) extends ScallopConf(args) {
     name = "file-data-redirect-base-url", short = 'b',
     descr = """Base URL from which redirect URLs to the file items in this dataset will be constructed""",
     default = None)
+  val stubAVfiles = opt[Boolean](
```

Comment:
have you considered using `toggle` rather than `opt`? Seems more fitting here. See also its documentation for an explanation.